src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -189,6 +191,8 @@ public byte[] GetDataBytes(bool forHash = false)
                     (Version >= 13 || !p.Name.Equals(nameof(UsePasswordManager))) &&
                     (Version >= 13 || !p.Name.Equals(nameof(SmSeats))) &&
                     (Version >= 13 || !p.Name.Equals(nameof(SmServiceAccounts))) &&
+                    // LimitCollectionCreationDeletion was added in Version 14
+                    (Version >= 14 || !p.Name.Equals(nameof(LimitCollectionCreationDeletion))) &&
```

Comment:
One problem with the current approach is that if the feature flag is off, the license file won't match the org object (because the create/update methods will choose to ignore the value in the license file). So there's a risk that validation will fail. Given that it's not a billing-critical property, maybe we just exclude it from validation for now.

Alternatively, maybe we don't flag any of this logic and we just let the value be updated from the license file. After all, the actual logic will still be flagged on self-hosted. My concern there was that it'll be a latent change that will suddenly take effect after updating the self-hosted instance, and that could be unexpected.

There's a bit to this so let's discuss in the next FC sync.

Reply:
@vincentsalucci I think the conclusion from our sync was that my feedback here is fine. Amended the comment above.